test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,573 @@
+
+	// we have to manually construct the subnet group names for rds and elasticache,
+	// since tag filtering isnt currently available
+	name := resources.ShortenString(fmt.Sprintf("%s-%s", clusterTag, "subnet-group"), 40)
+
+	// build array list of all vpc private subnets
+	var subnetIDs []*string
+	for _, subnet := range subnets {
+		subnetIDs = append(subnetIDs, subnet.SubnetId)
+	}
+
+	// verify rds subnet groups
+	rdsSvc := rds.New(session)
+	err = verifyRdsSubnetGroups(rdsSvc, name, subnetIDs)
+	testErrors.rdsSubnetGroupsError = err.(*networkConfigTestError).rdsSubnetGroupsError
+
+	// verify elasticache subnet groups
+	cacheSvc := elasticache.New(session)
+	err = verifyCacheSubnetGroups(cacheSvc, name, subnetIDs)
+	testErrors.rdsSubnetGroupsError = err.(*networkConfigTestError).rdsSubnetGroupsError
+
+	// peering connection and route tables
+	if vpc == nil {
+		testErrors.peeringConnError = []string{"skipping peering connection test, vpc cannot be nil"}
+		testErrors.standaloneRouteTableError = []string{"skipping standalone route table test, vpc cannot be nil"}
+		testErrors.clusterRouteTablesError = []string{"skipping cluster route table test, vpc cannot be nil"}
+	} else {
+		conn, err := verifyPeeringConnection(ec2Sess, clusterTag, expectedCidr, aws.StringValue(vpc.VpcId))
+		testErrors.peeringConnError = err.(*networkConfigTestError).peeringConnError
+
+		err = verifyStandaloneRouteTable(ec2Sess, clusterTag, conn)
+		testErrors.standaloneRouteTableError = err.(*networkConfigTestError).peeringConnError
+
+		err = verifyClusterRouteTables(ec2Sess, clusterTag, expectedCidr, conn)
+		testErrors.clusterRouteTablesError = err.(*networkConfigTestError).clusterRouteTablesError
+	}
+
+	if testErrors.isValid() {
+		t.Fatal(testErrors.Error())
+	}
+}
+
+// verify that the standalone vpc is created
+func verifyVpc(session *ec2.EC2, clusterTag, expectedCidr string) (*ec2.Vpc, error) {
+	newErr := &networkConfigTestError{
+		vpcError: []string{},
+	}
+
+	// filter vpcs by name and integreatly tag
+	describeVpcs, err := session.DescribeVpcs(&ec2.DescribeVpcsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(vpcTagName)},
+			},
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("could not find vpc: %v", err))
+		return nil, newErr
+	}
+
+	// only one vpc is expected
+	vpcs := describeVpcs.Vpcs
+	if len(vpcs) != 1 {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("expected 1 vpc but found %d", len(vpcs)))
+		return nil, newErr
+	}
+
+	// cidr blocks should match
+	vpc := vpcs[0]
+	foundCidr := aws.StringValue(vpc.CidrBlock)
+	if foundCidr != expectedCidr {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("cidr blocks not equal, expected %s but got %s", expectedCidr, foundCidr))
```

Comment:
Should we expand that error message to explain what the cidr block should equal too. Or maybe some extra developer docs to call out we expect the cidr block of the vpc to be equal to the cidr block from the strategy map.